pkg/reconciler/workload/namespace/namespace_reconcile.go

```diff
@@ -40,12 +40,13 @@ import (
 
 	tenancyv1alpha1 "github.com/kcp-dev/kcp/pkg/apis/tenancy/v1alpha1"
 	workloadv1alpha1 "github.com/kcp-dev/kcp/pkg/apis/workload/v1alpha1"
+	"github.com/kcp-dev/kcp/pkg/syncer"
 	conditionsapi "github.com/kcp-dev/kcp/third_party/conditions/apis/conditions/v1alpha1"
 	"github.com/kcp-dev/kcp/third_party/conditions/util/conditions"
 )
 
 const (
-	ClusterLabel            = "workloads.kcp.dev/cluster"
+	ScheduledClusterLabel   = "workloads.kcp.dev/scheduled-cluster"
```

Comment:
am confused. What is this now after we added the new labels?

Reply:
done!